_data/subscription-benefits.yml

```diff
@@ -34,7 +20,12 @@
         </div>
     </div>
 
-- title: Go through a production readiness review.
+- title: Get automated infrastructure updates<sup>(Beta)</sup>
+  description: |
+    <p>Everything in the DevOps world is constantly changing: AWS releases hundreds of new features a year, Terraform releases new versions every two weeks, security vulnerabilities are found daily, and we update the Infrastruture as Code Library all the time. As a Gruntwork subscriber, you can keep up to date with all of these changes—automatically—by using Patcher<sup>(beta)</sup>. Patcher is a tool that can automatically scan code in supported languages (currently, Terraform and Packer), identify updates that need to be applied, apply those updates by transforming the code, and submit a pull request for you to review.</p>
+    <p class="text-muted"><em>Patcher is currently in beta and available upon request with certain limitations.</em></p>
+
+- title: Go through a production readiness review
```

Comment:
I find this a little confusing. It's a one-time service, but included on the support page? Do all support programs include it, or just enterprise? Is there a fee for it, or is it included with support? This one again makes me think that a Consulting section is ideal, though I understand your concerns around it. Let's discuss offline.

Reply:
Agreed, but not something we can fix now. Also, I again want to stress that consulting is NOT what we should be aiming for. Aim for leverage/reuse/recurring revenue in all offerings. That's why PRR is in support as of now.

Reply:
Ok, I think at minimum we should add something like "(Enterprise Plan only)" so people don't feel duped.

Reply:
Regarding consulting, I agree with your general position, but have a different take on how we negotiate it. Queued for discussion later on.

Reply:
Added a note about Enterprise plan: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/23/commits/be7c97ecf6a0f672ac8fd1ade84ac9de8eabc84e